integration-cli/docker_cli_run_unix_test.go

```diff
@@ -299,7 +299,10 @@ func (s *DockerSuite) TestRunEchoStdoutWithMemoryLimit(c *check.C) {
 	}
 }
 
-// should run without memory swap
+// TestRunWithoutMemoryswapLimit should run without memory swap limit.
```

Comment:
@sunyuan3 yes exactly, the one explaining how it works is fine and you can leave it